web/test/step_definitions/browser.js

```diff
@@ -1,11 +1,16 @@
 import { Given, Then } from 'cypress-cucumber-preprocessor/steps';
 
-const url = 'http://localhost:8080';
+const url = 'http://localhost:4200';
 
 Given(/^the "([^"]*)" page is open$/, (path) => {
     cy.visit(`${url}/${path}`);
 });
 
 Then(/^the title on the page says "([^"]*)"$/, (check) => {
-    cy.title().should('include', check);
+  cy.title().should('include', check);
 });
+
+And(/^the text "([^"]*)" is in the element "([^"]*)"$/, (text, element) => {
```

Comment:
There is no `And` import, this should be `Then`, but in the `.feature` file, `And` can be used

Reply:
okay . I will update this.